resources/views/book_detail.blade.php

```diff
@@ -0,0 +1,178 @@
+                                <input type="hidden" name="cancel_return" value="" />
+                                <input type="submit" name="submit" value="{{ trans('home.add_to_form') }}" class="button" />
+                            </fieldset>
+                        </form>
+                    </div>
+
+                </div>
+
+                <ul class="social-nav model-3d-0 footer-social w3_agile_social single_page_w3ls">
+                    <li class="share">{{ trans('home.share') }}</li>
+                    <li>
+                        <a href="" class="facebook">
+                            <div class="front"><i class="fa fa-facebook" aria-hidden="true"></i></div>
+                            <div class="back"><i class="fa fa-facebook" aria-hidden="true"></i></div>
+                        </a>
+                    </li>
+
+                    <li>
+                        <a href="" class="twitter">
+                            <div class="front"><i class="fa fa-twitter" aria-hidden="true"></i></div>
+                            <div class="back"><i class="fa fa-twitter" aria-hidden="true"></i></div>
+                        </a>
+                    </li>
+
+                    <li>
+                        <a href="" class="instagram">
+                            <div class="front"><i class="fa fa-instagram" aria-hidden="true"></i></div>
+                            <div class="back"><i class="fa fa-instagram" aria-hidden="true"></i></div>
+                        </a>
+                    </li>
+
+                    <li>
+                        <a href="" class="pinterest">
+                            <div class="front"><i class="fa fa-linkedin" aria-hidden="true"></i></div>
+                            <div class="back"><i class="fa fa-linkedin" aria-hidden="true"></i></div>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+
+            <div class="clearfix"> </div>
+
+            <div class="responsive_tabs_agileits">
+                <div id="horizontalTab">
+                    <ul class="resp-tabs-list">
+                        <li>{{ trans('message.description') }}</li>
+                        <li>{{ trans('home.reviews') }}</li>
+                    </ul>
+                    <div class="resp-tabs-container">
+                        <div class="tab1">
+                            <div class="single_page_agile_its_w3ls">
+                                <p>{{ $book->description }}</p>
+                            </div>
+                        </div>
+
+                        <div class="tab2">
+                            {{-- comment book --}}
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="w3_agile_latest_arrivals">
+                <h3 class="wthree_text_info">{{ trans('home.related_books') }}</h3>
+                @foreach ($related_books as $related_book)
```

Comment:
```suggestion
                @foreach ($relatedBooks as $relatedBook)
```
đặt lại tên các biến